sympy/series/__init__.py

```diff
@@ -5,7 +5,6 @@
 from .gruntz import gruntz
 from .series import series
 from .residues import residue
+from .sequences import (EmptySequence, SeqPer, SeqFormula, SeqFunc)
 
 O = Order
-
-__all__ = ['gruntz', 'limit', 'series', 'O', 'Order', 'Limit', "residue"]
```

Comment:
Why did you delete this, instead of adding your stuff to it? This controls what's actually imported from the submodules.